view/statistics-files-completed.js

```diff
@@ -148,4 +148,13 @@ exports['statistics-main'] = function () {
 				});
 			})
 		));
+
+	var checkExist = setInterval(function () {
```

Comment:
Let's make a generic component out of this - separate module which exports function. The function should accept one param - id of the table around which we will build the sorting mechanism.